GoodMorning/GoodMorning/HomeScene/Views/TodayLuckStackView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  TodayLuckStackView.swift
+//  GoodMorning
+//
+//  Created by Sunny on 2023/08/10.
+//
+
+import UIKit
+
+final class TodayLuckStackView: UIStackView {
+
+    private let dobleLabel = DoubleLabelStackView(type: .luck)
+
+    private let imageView: UIImageView = {
+        let imageView = UIImageView(image: Image.luck.image)
+        imageView.widthAnchor.constraint(equalTo: imageView.heightAnchor).isActive = true
```

Comment:
```suggestion
private let luckyImageIcon: UIImage = UIImage(systemName: "bubbles.and.sparkles.fill")
    private let imageView: UIImageView = {
        let imageView = UIImageView(image: luckyImageIcon)
        imageView.widthAnchor.constraint(equalTo: imageView.heightAnchor).isActive = true
        imageView.tintColor = .black
        return imageView
    }()
```
현재 `TodayLuckStackView`에서는 하나의 Icon Image만 활용하는 데 case를 나눌 수 있는 Enum 타입으로 구현하신 이유가 궁금합니다! 
만약 다른 곳에서도 사용할 수 있는 Icon들을 타입으로 구현하고 싶으면,  Enum IconImage 타입으로 프로젝트에서 사용하는 모든 Icon Image를 처리할 수 있을 거 같습니다! 혹시 Sunny는 해당 타입에서만 활용할 수 있는 Image 타입을 Enum으로 구현하신 이유가 있을까요?🤔

Reply:
아! 저는 UIImageView.image에 매직 스트링으로 넣어 주는 것을 피하기 위해 해당 파일에서만 사용할 수 있는 enum 타입을 만들었어요!
근데 미로의 피드백을 받고 나니깐 enum을 활용한 name space를 만들던지, fileprivate를 활용한 UIImage extension을 사용할 수도 있을 것 같아요. 

미로의 생각에는 어떤 방법이 더 나아 보이나요?

Reply:
두 가지 방법 다 좋은 거 같은데, enum을 활용하는 것이 좋을 거 같습니다! 날씨 앱에서 또 SFSymbol을 활용할 일이 있을 거 같아서요!☺️